src/highdicom/seg/sop.py

```diff
@@ -287,6 +288,8 @@ def __init__(
                 'Only one source image should be provided in case images '
                 'are multi-frame images.'
             )
+        is_relative_to_slide = hasattr(src_img, 'ImageOrientationSlide')
```

Comment:
I think this is basically just a duplicate of the logic in line 334. We probably don't want two local variables capturing the same information floating around, it's bound to cause future issues.

```python
        if (hasattr(src_img, 'ImageOrientationSlide') or
                hasattr(src_img, 'ImageCenterPointCoordinatesSequence')):
            self._coordinate_system = CoordinateSystemNames.SLIDE
        else:
            self._coordinate_system = CoordinateSystemNames.PATIENT
```

Reply:
True! Addressed via 20787eb877effa0e04e165e5c70283c36eb6f345